File: LAB 19/Task4.py
def print_students(students):
    print("\nStudent List:")
    for student in students:
        print(student)

# List to store student names
students = []

while True:
    name = input("Enter student name (or press Enter to finish): ")
    if name == "":
        break
    students.append(name)

# Call the function to print the students
print_students(students)

File: LAB 19/Task4.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let students = [];

function askName() {
    rl.question("Enter student name (or press Enter to finish): ", function(name) {
        if (name === "") {
            // Finished entering names, print the list
            console.log("\nStudent List:");
            students.forEach(student => console.log(student));
            rl.close();
        } else {
            students.push(name);
            askName(); // Ask next name
        }
    });
}
askName();
